packages/massa-web3/src/experimental/basicElements/internal.ts

```diff
@@ -17,11 +17,12 @@ export function extractData(
   return extractedData
 }
 
-export function checkPrefix(str: string, ...expected: string[]): void {
+export function mustExtractPrefix(str: string, ...expected: string[]): string {
```

Comment:
Exported function should have a documentation

Reply:
i aim to delete this function because its implemented with a shortcut (we arbitrary take the length of the first exected prefix), so if expected prefix length are differents, this will lead to wrong behaviour.
but yes ill add ts doc for now